values.yaml

```diff
@@ -162,18 +147,17 @@ healthlogger:
   requests:
     memory: 100Mi
     cpu: 0.1
-  annotations: {}
-  nodeSelector: {}
   watch_interval: 60
 
-# Deprecated fields
-# pipEnabled: true
-# pipHost: "http://pelias-pip-service:3102/"
-
-# interpolationEnabled: false
-# interpolationHost: "http://pelias-interpolation-service:3000/"
-
-# Importer settings
 whosonfirst:
-  sqlite: false
+  sqlite: true 
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo

Reply:
The import process will not work if we ignore this value.